.github/workflows/ci-test.yaml

```
@@ -397,6 +397,38 @@ jobs:
           find tests/ -name 'requirements.txt' -exec pip install -r {} \;
           pytest tests
 
+  unit-tests-fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Install go
+        run: sudo snap install go --classic --channel=latest/stable
+      - name: Fuzz tests
+        run: |
+          files=()
+          for test in $(find . -name '*_test.go'); do
+            if grep -q 'func.* Fuzz' "$test"; then
+              files+=("$test")
+            fi
+          done
+          echo "files found with fuzzing tests: ${fuzz_targets[@]}"
```

`fuzz_targets` was probably renamed to `files` (or the opposite)
```suggestion
          echo "files found with fuzzing tests: ${files[@]}"
```